hazelcast/src/test/java/com/hazelcast/test/mocknetwork/TestNodeRegistry.java

```diff
@@ -50,6 +51,15 @@ public NodeContext createNodeContext(Address address) {
         return new MockNodeContext(this, address);
     }
 
+    public NodeContext createNodeContext(Address address, Set<Address> initiallyBlockedAddresses) {
```

Comment:
Minor comment: This is mostly duplicate of `createNodeContext(Address address)`. Former one can delegate to this method;

```java
public NodeContext createNodeContext(Address address) {
    return createNodeContext(address, Collections.emptySet());
}
```